### PR TITLE
completed the modal component

### PR DESCRIPTION
**Pull request by therajatg**
Please review the modal component.

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
you can add blank attribute of all socil media link ! remaing looks good

File: docs/modal.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modal Component</title>
+    <link rel="stylesheet" href="/styles/main-style.css">
+    <link rel="stylesheet"
+    href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
```
Comment (shuntzhere) [inline docs/modal.html]: It's better to define stylesheets before the title.

**Review by shuntzhere — COMMENTED** [review]
Add descriptions of the work you have done, a video or a screenshot. Also, suggested a minor change. The rest looks neat.

File: docs/modal.html (hunk 1 above)
Comment (therajatg) [reply docs/modal.html]: I did not took this in consideration. Thanks for pointing it out, I defined stylesheets before the title.

 PS: This is the preview of the modal component- https://deploy-preview-14--parts-builder.netlify.app/docs/modal

**Review by shuntzhere — APPROVED** [review]
Looks clean, you can go ahead with the merge.

**Review by iamsaurav28 — APPROVED** [review]
everything looks good , still check once again if u have forgot to add any component.

**Pull request merged**